dor/providers/parsers.py

```diff
@@ -22,17 +26,18 @@ def get_type(self):
         hdr = self.tree.find("METS:metsHdr")
         return hdr.get("TYPE")
 
-    def get_alternate_identifier(self):
+    def get_alternate_identifier(self)-> AlternateIdentifier:
         alt_record_id = self.tree.find("METS:metsHdr/METS:altRecordID")
         if alt_record_id:
             return AlternateIdentifier(
                 type=alt_record_id.get("TYPE"), id=alt_record_id.text
             )
+        return AlternateIdentifier(type="", id="")    
```

Comment:
I think this can be simplified, since `tree.find` should throw an error if the element isn't found.